packages/client/components/ReviewRequestToJoinOrgModal.tsx

```diff
@@ -0,0 +1,117 @@
+import graphql from 'babel-plugin-relay/macro'
+import React, {useState, Suspense} from 'react'
+import {PreloadedQuery, usePreloadedQuery} from 'react-relay'
+import DialogContainer from './DialogContainer'
+import DialogContent from './DialogContent'
+import DialogTitle from './DialogTitle'
+import PrimaryButton from './PrimaryButton'
+import SecondaryButton from './SecondaryButton'
+import {ReviewRequestToJoinOrgModalQuery} from '../__generated__/ReviewRequestToJoinOrgModalQuery.graphql'
+import useQueryLoaderNow from '../hooks/useQueryLoaderNow'
+import Checkbox from './Checkbox'
+
+const query = graphql`
+  query ReviewRequestToJoinOrgModalQuery {
+    viewer {
+      teams {
```

Comment:
-1 We're querying all teams, not just the relevant ones. I see that all of them probably have the matching domain, but I think it should be filtered at least to orgs with the feature flag set.
We might even just limit it to the orgId from the request.